python_transport/wirepas_gateway/dbus/sink_manager.py

```diff
@@ -244,7 +259,13 @@ def write_config(self, config):
             new_state = stack_started
 
         try:
-            self.proxy.SetStackState(new_state)
+            current_state = (self.proxy.StackStatus & 0x01) == 0
```

Comment:
What is the reason behind the and with 0x01?

Reply:
It's because StackStatus is a bitfield with bit 0 being StackRunning(0) or stackStopped(1).
So current_state is True if started and false otherwise

Reply:
Thanks!